{
  "name": "@hidoo/gulp-util-error-handler",
  "version": "0.18.0",
  "description": "Utility that handle task error for gulp.",
  "engines": {
    "node": ">=10.0.0"
  },
  "main": "lib/index.js",
  "files": [
    "lib/"
  ],
  "scripts": {
    "prebuild": "rimraf ./lib",
    "build": "babel ./src --out-dir ./lib",
    "test": "cross-env NODE_ENV=test mocha"
  },
  "keywords": [
    "gulp"
  ],
  "publishConfig": {
    "access": "public"
  },
  "author": "hidoo",
  "license": "MIT",
  "dependencies": {
    "chalk": "4.1.0",
    "fancy-log": "1.3.3"
  },
  "devDependencies": {
    "@babel/cli": "7.13.14",
    "@babel/core": "7.13.14",
    "@babel/preset-env": "7.13.12",
    "@babel/register": "7.13.14",
    "babel-preset-power-assert": "3.0.0",
    "cross-env": "7.0.3",
    "mocha": "8.3.2",
    "power-assert": "1.6.1",
    "rimraf": "3.0.2",
    "sinon": "10.0.0",
    "strip-ansi": "6.0.0"
  }
}
